### PR TITLE
features updated

### PR DESCRIPTION
**Pull request by akhilsingh-git**
- Updated mssql-scaffolder.component.html 
- Updated mssql-scaffolder.component.ts

### CODE REVIEW

File: Frontend/src/app/mssql-scaffolder/mssql-scaffolder.component.ts [hunk 1]
```diff
@@ -305,16 +334,3 @@ enum ConnectionStatus {
   connecting,
   connected
 }
-
```
Comment (IPdotSetAF) [inline Frontend/src/app/mssql-scaffolder/mssql-scaffolder.component.ts:308]: The interfaces provide a little bit of documentation and IntelliSense help.

**Review by IPdotSetAF — CHANGES_REQUESTED** [review]
so far so good.
but the html file seems unchanged or maybe you forgot to commit it, please check it out.

File: Frontend/src/app/mssql-scaffolder/mssql-scaffolder.component.ts [hunk 2]
```diff
@@ -305,16 +334,3 @@ enum ConnectionStatus {
...
-  server: AbstractControl<string>;
-  username: AbstractControl<string>;
-  password: AbstractControl<string>;
-}
-
-interface ScaffoldForm {
-  database: AbstractControl<string>;
-  schema: AbstractControl<string>;
-  isTable: AbstractControl<boolean>;
-}
```
Comment (IPdotSetAF) [inline Frontend/src/app/mssql-scaffolder/mssql-scaffolder.component.ts:320]: The interfaces provide a little bit of documentation and IntelliSense help.

**Review by IPdotSetAF — DISMISSED** [review]

**Comment by akhilsingh-git** [comment]
Updated now

**Comment by IPdotSetAF** [comment]
these are the controls(select) that need to be improved:

image

the search box should looks like a simple select tag but also accept a text input (which is used to filter the options inside).
after each keydown event, it should refresh the list and show the select menu.

the search box and input should not show as different fields on the ui

the search button on the side is optional though.

**Pull request closed without merging**